pkg/kubeapiserver/authorizer/modes/modes.go

```diff
@@ -29,10 +31,5 @@ var AuthorizationModeChoices = []string{ModeAlwaysAllow, ModeAlwaysDeny, ModeABA
 
 // IsValidAuthorizationMode returns true if the given authorization mode is a valid one for the apiserver
 func IsValidAuthorizationMode(authzMode string) bool {
-	for _, validMode := range AuthorizationModeChoices {
-		if authzMode == validMode {
-			return true
-		}
-	}
-	return false
+	return sets.NewString(AuthorizationModeChoices...).Has(authzMode)
```

Comment:
This is much heavier than the simple loop. Don't think it's a good idea.

Reply:
Agree, but this is not called frequently enough to matter either way

Reply:
@sttts @liggitt Thanks for review, I think it has better readability for using `sets`.

Reply:
@sttts @liggitt  Thanks for review, I think it has better readability for using `sets`.